WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -64,11 +68,11 @@ final class CollectOrderPaymentUseCase: NSObject, CollectOrderPaymentProtocol {
 
     /// Alerts presenter: alerts from the various parts of the payment process are forwarded here
     ///
-    private let alertsPresenter: CardPresentPaymentAlertsPresenting
+    private let alertsPresenter: any CardPresentPaymentAlertsPresenting<AlertPresenter.AlertDetails>
```

Comment:
might be able to `typealias AlertDetails = AlertPresenter.AlertDetails` and use `AlertDetails` in this file to make things look shorter

Reply:
Yep. We also might just be able to use the `AlertPresenter` typealias directly, I did start doing that in some of the later PRs